client/components/mainImage.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MainImg = styled.img`
+  vertical-align: baseline;
+  width: 600px;
+  heigth: 600px;
+  margin-top: 15px;
+`;
+
+const MainImage = ({url}) => (
+  <MainImg src={url} alt="I'm the big one!" />
+);
```

Comment:
I was thinking about Laurie Voss' talk about accessibility and the alt tag. It would be cool to get a tag here that matched the game, view or pic being displayed. Probably a lot to ask with dummy data especially.

Reply:
That's a good point.  Long-run, adding an alt field to the images table would be great!